README.md

```diff
@@ -1,250 +1,326 @@
-You can find the actual firmware images generated by the ImageBuilder (and the ImageBuilder itself)
-in `firmwares`. The layout looks like the following:
 
+### Directory Layout of `firmwares/`
+
+You can find the actual firmware images generated by the ImageBuilder (and the
+ImageBuilder itself) in `firmwares/`. The layout looks like the following:
+
+``` 
+firmwares/ 
+  TARGET/ 
+    backbone/ 
+      images.. 
+    default/
+      images..
+    ...
+    OpenWrt-ImageBuilder-....tar.xz 
+    OpenWrt-SDK-....tar.xz
+    initrd/ 
+      images..
+    packages/ 
+      packages/<ARCH>
+        base/*.ipk 
+        luci/*.ipk
+        packages/*.ipk
+        packages_berlin/*.ipk
+        routing/*.ipk
+      targets/MAINTARGET/SUBTARGET/packages/
+          *.ipk
 ```
-firmwares/
-    TARGET/
-        backbone/
-           images..
-        default/
-           images..
-        ...
-        OpenWrt-ImageBuilder-....tar.xz
-        OpenWrt-SDK-....tar.xz
-        initrd/
-           images..
-        packages/
-           packages/<ARCH>
-              base/*.ipk
-              luci/*.ipk
-              packages/*.ipk
-              packages_berlin/*.ipk
-              routing/*.ipk
-           targets/MAINTARGET/SUBTARGET/packages/
-              *.ipk
-```
 
-As you notice there are several different image variants ("backbone", "default", etc.).
-These different *packages lists* are defined in `packages/`.
-See the "Features" section above for a description of the purpose of each package list.
-With the "OpenWrt-Imagebuilder" you can assemble your own image variant with your
-*packages lists* without having to compile everything yourself. The "OpenWrt-SDK" is
-the fastest way to build your own packages or programs without compiling OpenWrt itself.
-The "initrd" directory contains some initrd-images for netboot, which are required on
-some boards to initially install OpenWrt.
+As you notice there are several different image variants ("backbone", "default",
+etc.). 
++ These different *packages lists* are defined in `packages/`. See the
+    [wiki](https://wiki.freifunk.net/Berlin:Firmware#Image_Types_.28English.29)
+    for a description of the purpose of each package list.
++ With the "OpenWrt-Imagebuilder" you can assemble your own image variant with
+    your *packages lists* without having to compile everything yourself. The
+    *OpenWrt-SDK* is the fastest way to build your own packages or programs 
+    without compiling OpenWrt itself.
++ The *initrd* directory contains some initrd-images for netboot, which is
+    required on some boards to initially install OpenWrt.
+
 
 ### customizing make
 
 `make` will use by default `TARGET` and `PACKAGES_LIST_DEFAULT` defined in
 `config.mk`. You can customize this by overriding them:
 
-```
-make TARGET=mpc85xx PACKAGES_LIST_DEFAULT=backbone
-```
-in addition you can build your own image from a prebuilt imagebuilder by something like:
+``` make TARGET=mpc85xx PACKAGES_LIST_DEFAULT=backbone ```
```

Comment:
As you touch it: 
* Since some time it's mpc85xx-generic
* Should we use the changed packagelists here (notunnel, manual, ...)?

Reply:
Adjusted to new naming.